docs/user-guide/howto.rst

```diff
@@ -233,3 +233,28 @@ to your code:
     pbar.SHOW_PROGRESS_BAR = True
 
 .. accordion-footer::
+
+.. accordion-header::
+    :id: collapseHowToSixteen
+    :title: Colour blind friendly visualisations
```

Comment:
Thanks. Fixed

I was answering to a specific issue. If one wishes a general HowTo about the plotting style, it might be an other PR. As a compromise, I can refer to this link at this end of this HowTo.

We discuss about the list of 'apparent' commits during the last dev call. As you can see, only the changed file are in reality committed. This list is due to the fact that I did my 'rebase' via the GitHub web site (and not with a local git command line).

Reply:
> I was answering to a specific issue. If one wishes a general HowTo about the plotting style, it might be an other PR. As a compromise, I can refer to this link at this end of this HowTo.

OK I see. But I suppose it is could useful to give the general info e.g. How to change plotting style, compared to the specific issue of color-blind visualizations. It could be done in the same how to.

Note that to be really color-blind friendly the example you show would require to use different line styles which we never really do in our `peek` and `plot` methods.

Reply:
For the line style, the CTAO recommendations do not mention that. Do you have then an other reference? With such info, a much more complex PR is needed. It is important to know then...

Is your proposal to change from an 'Color-blind friendly visualisations' HowTo for an 'Changing plotting style' HowTo that will include the Color-blind friendly visualisations? If yes, I will change the PR (and its title) accordingly...

Reply:
> For the line style, the CTAO recommendations do not mention that. Do you have then an other reference? With such info, a much more complex PR is needed. It is important to know then...

Indeed. I find the color blind line plots hard to read once more than 3 colors are used because of the proximity of the gray colors. Probably this means that line styles should differ as well. This is definitely a more complex PR, and it is not sure we need it.

Reply:
> Is your proposal to change from an 'Color-blind friendly visualisations' HowTo for an 'Changing plotting style' HowTo that will include the Color-blind friendly visualisations? If yes, I will change the PR (and its title) accordingly...

Maybe one can simply modify the title to e.g. 'Changing plotting style and color-blind friendly visualizations', mention that matplotlib has a global setting to set predefined plotting style that can be used in particular to switch to e.g. color blind one and add the link to the list other styles at the end?

Reply:
done